packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -128,6 +128,30 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
   Future<void> setWebOptions(int playerId, VideoPlayerWebOptions options) {
     throw UnimplementedError('setWebOptions() has not been implemented.');
   }
+
+  /// Gets the available audio tracks for the video.
+  Future<List<VideoAudioTrack>> getAudioTracks(int playerId) {
```

Comment:
There needs to be a method to let the app-facing package know whether it's safe to call audio track methods. https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#api-support-queries